Documentation/StyleGuide.md

```diff
@@ -110,8 +141,38 @@ reader are not expected to know each other personally, so avoid overly familiar
 terms or colloquialisms such as "let's."
 
 When writing specific instructions that a reader must follow exactly in order to
-accomplish a task, use the second person ("you", "your", etc.) Otherwise, avoid
-using the second person.
+accomplish a task, use the second person ("you", "your", etc.).
+
+#### Articles
+
+When writing articles or curating content, keep the structure simple and
+relatively flat. Place articles in topic groups alongside the symbols.
+
+Some basic rules to follow when creating an article are:
+- Begin an article title with a *gerund* (a verb ending in "ing").
```

Comment:
```suggestion
- Begin an article title with a _gerund_ (a verb ending in "ing").
```

Some Markdown renderers render `*foo*` as **foo** instead of _foo_.